api/application.go

```diff
@@ -358,6 +358,30 @@ func (a *ApplicationAPI) UploadApplicationImage(ctx *gin.Context) {
 	})
 }
 
+func (a *ApplicationAPI) RemoveApplicationImage(ctx *gin.Context) {
+	withID(ctx, "id", func(id uint) {
+		app, err := a.DB.GetApplicationByID(id)
+		if success := successOrAbort(ctx, 500, err); !success {
+			return
+		}
+		if app != nil && app.UserID == auth.GetUserID(ctx) {
+			if app.Image == "" {
+				ctx.AbortWithError(400, fmt.Errorf("app with id %d does not have a customized image", id))
```

Comment:
> I am also not sure whether an error message makes sense, in case there is no image. IMO it should be a warning or info message, but not sure how to generate such a type.

I'm fine with 400, the UI shouldn't allow deleting an image when there isn't one.